af/shovel/check_sanitised.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python2
+
+# That's a sanity check to ensure that we can migrate from historical public/sanitised to public/autoclaved:
+# 1. every sanitised/ file is in autoclaved/
+# 2. every sanitised/ file almost equals corresponding autoclaved/
+# 3. DOES NOT check that every autoclaved/ is in sanitised/ (that's not true as some reports-raw were never sanitised)
+#
+# This sanity check means that sanitised/ is safe to delete.
+
+import argparse
+import datetime
+import hashlib
+import os
+import tempfile
+from functools import partial
+
+import ujson
+
+import autoclaving
+from canning import dirname
+from centrifugation import stream_datum, FILE_START, FILE_END, REPORT_START, REPORT_END, BADBLOB, DATUM
```

Comment:
`stream_datum` is not defined in centrifugation (anymore?) and is causing the `sanitised_check` task to fail in the `hist_canning` dag.

see:

```
[2017-08-24 00:23:10,171] {base_task_runner.py:95} INFO - Subtask: [2017-08-24 00:23:10,170] {bash_operator.py:94} INFO - File "/usr/local/bin/check_sanitised.py", line 21, in <module>
[2017-08-24 00:23:10,171] {base_task_runner.py:95} INFO - Subtask: [2017-08-24 00:23:10,170] {bash_operator.py:94} INFO - from centrifugation import stream_datum, FILE_START, FILE_END, REPORT_START, REPORT_END, BADBLOB, DATUM
[2017-08-24 00:23:10,171] {base_task_runner.py:95} INFO - Subtask: [2017-08-24 00:23:10,171] {bash_operator.py:94} INFO - ImportError: cannot import name stream_datum
[2017-08-24 00:23:11,888] {base_task_runner.py:95} INFO - Subtask: [2017-08-24 00:23:11,888] {bash_operator.py:97} INFO - Command exited with return code 1
```